Source/NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationBallsBounce.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  NVActivityIndicatorAnimationBallsBounce.swift
+//  NVActivityIndicatorView
+//
+// The MIT License (MIT)
+
+// Copyright (c) 2016 Vinh Nguyen
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+//
+
+import UIKit
+
+class NVActivityIndicatorAnimationBallsBounce: NVActivityIndicatorAnimationDelegate {
+
+    func setUpAnimation(in layer: CALayer, size: CGSize, color: UIColor) {
+        for index in (0...1) {
+            bouncingBall(in: layer, size: size, color: color, startingAt: CACurrentMediaTime() + Double(-1 * index))
```

Comment:
It's negligible, but I think both balls should have the same base starting time.

Reply:
I tried it but it produces a single ball bouncing.

Reply:
How about plus a positive value instead?

Reply:
Ok positive values added but by playing with numbers you will see negative ones on scale animation but it produces a better animation INMO.